Make.versions

```diff
@@ -48,8 +48,8 @@
 #   line changed in git).
 #
 
-IOS_PACKAGE_VERSION=14.4.1.$(IOS_COMMIT_DISTANCE)
-MAC_PACKAGE_VERSION=6.99.0.$(MAC_COMMIT_DISTANCE)
+IOS_PACKAGE_VERSION=14.4.0.$(IOS_COMMIT_DISTANCE)
```

Comment:
no, you can't downgrade - it will confuse tools/people
make that `14.4.2`

Reply:
and since that's a commit, the distance will be an issue unless you also modify the MAC_PACKAGE_VERSION line